PublicLedger.test.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import fetchMock from 'jest-fetch-mock';
+import { Template } from '@daml/types';
+import Ledger from '@daml/ledger';
+import React, { ComponentType } from 'react';
+import { sign } from 'jsonwebtoken';
+import { renderHook, RenderHookResult } from '@testing-library/react-hooks';
+import { PublicLedger, usePartyAsPublic, useLedgerAsPublic, useQueryAsPublic } from './index';
+
+const TEST_LEDGER_ID ='test-ledger-id'
+const PUBLIC_PARTY ='PUBLIC'
```

Comment:
This is `Public` instead of all caps

Reply:
Sure, on DABL. But this is just a testing constant.